Makefile

```diff
@@ -23,7 +23,7 @@ clean:
 
 .PHONY: test
 test:
-	go test -short -v -shuffle=on -count=1 -race -timeout 20m ./...
```

Comment:
This [enables non-sim tests](https://github.com/filecoin-project/mir/blob/6ed94fdef106ad3f36a5ced0d149bacfaebb094d/pkg/systems/smr/smr_test.go#L213-L216).

Reply:
Fixed - https://github.com/filecoin-project/mir/pull/290/commits/25c8f7d27247f34a68ad7d61ee4c8930294712c9

Reply:
@sergefdrv please review the fix. I just reverted the change.